Branch.podspec

```diff
@@ -15,26 +15,31 @@ Use the Branch SDK (branch.io) to create and power the links that point back to
   s.homepage         = "https://branch.io"
   s.license          = 'Proprietary'
   s.author           = { "Branch" => "support@branch.io" }
-  s.source           = { :git => "https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK.git", :tag => s.version.to_s }
+  s.source           = { git: "https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK.git", tag: s.version.to_s }
   s.platform         = :ios, '7.0'
   s.requires_arc     = true
 
+  source_files =
```

Comment:
I'm pretty sure this can just be `s.source_files = ...`. https://guides.cocoapods.org/syntax/podspec.html#subspec That attribute should be inherited by all subspecs unless overridden.